commands/cloudapi-v6/nic.go

```diff
@@ -1033,3 +1044,151 @@ func DetachAllNics(c *core.CommandConfig) error {
 	fmt.Fprintf(c.Command.Command.OutOrStdout(), jsontabwriter.GenerateLogOutput("Nics successfully detached from Load Balancer"))
 	return nil
 }
+
+func validateIPv6CidrBlockForNIC(cidr string, lan ionoscloud.Lan) error {
```

Comment:
The only difference between `validateIPv6CidrBlockForLAN` and `validateIPv6CidrBlockForNIC` is the mask size, `64` vs `80` - perhaps pass this expected mask size as a parameter?

```
validateIPv6CidrBlock(cidr string, lan ionoscloud.Lan, expectedMask int) error

validateIPv6CidrBlockForNIC(...) {
   return validateIPv6CidrBlock(..., 80)
}

validateIPv6CidrBlockForLAN(...) {
   return validateIPv6CidrBlock(..., 64)
}
```

Reply:
(to reduce code duplication)